model/registry/host.yaml

```diff
@@ -111,10 +111,10 @@ groups:
           Model designation of the processor.
         examples: [ '11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz' ]
       - id: cpu.stepping
-        type: int
+        type: string
```

Comment:
Please add a changelog entry, particularly for this breaking change.

Reply:
For some reason I forgot to push it 🤦🏽‍♂️ . Thanks!